training_hyperparameter.yaml

```
@@ -0,0 +1,21 @@
+#model-stub and model_save_name will be selected in the autoencoder_training.py file
+# You can run the training with 'python autoencoder_training.py --name <NameAutoencoder> --model_path <path.to.model>
+# Simplifies the training of multiple models using a script
+
+data_params:
+  training_filepath: "data/heist/training_samples.npy"
+  test_filepath: "data/heist/test_samples.npy"
```

Wenn man die data generation skripte ausführt werden die Daten im `data_generation` Ordner gespeichert

```suggestion
  training_filepath: "data_generation/data/heist/training_samples.npy"
  test_filepath: "data_generation/data/heist/test_samples.npy"
```